### ISSUE TITLE
Making child surface size a percentage/relative to the parent

### ISSUE THREAD

**Issue by osiloke** [Famous/famous-angular#243]
New to famous-angular, I was wondering how i would go about making child surface sizes relative to the parent. I looked at existing issues but none seems to resolve to how this should be done in an angular way, something that has to do with data binding. The only option i can mage is that i create the surfaces in the controller and then render using the fa-render-node directive. I just want to find out if there is a better way of doing this.

Thanks

Basically, i would like to do something like the following

```
fa-app.full-screen
    fa-view
        fa-modifier(fa-size='[undefined, undefined]', fa-origin='[.5, .5]', fa-align='[0.5, 0.5]') 
            fa-surface(id="back", fa-background-color="'black'")
        fa-modifier(fa-align='[0.5, 0.5]', fa-origin='[0.5, 0.5]', fa-size='[0.5 * back.width, 0.5 * back.height]') 
            fa-surface(fa-background-color="'red'", id="bottom")
```
